ethcontract-common/Cargo.toml

```diff
@@ -19,4 +19,4 @@ serde_derive = "1.0"
 serde_json = "1.0"
 thiserror = "1.0"
 tiny-keccak = { version = "2.0", features = ["keccak"] }
-web3 = { version = "0.17", default-features = false }
+web3 = { git = "https://github.com/tomusdrw/rust-web3.git" }
```

Comment:
How long do we expect to have to depend on this git crate? What specific feature does this version have that isn't already available in v0.17?

Reply:
We need this one to be released:
https://github.com/tomusdrw/rust-web3/commit/2e2d1dfa906aa2dff6312b97d0d5ba804e9c8c6e